sass/index.scss

```diff
@@ -512,6 +491,34 @@ footer {
     section {
         padding: 100px 3.5rem 0px; // Set up to avoid nav overlap when linked
     }
+
+    // About
+    #about {
+        #about-cards {
+            flex-direction: column;
+
+            .card {
+                width: 100%;
+
+                &:not(.card:last-of-type) {
```

Comment:
Is this needed anymore?

Reply:
Removing it makes the two cards press up against each other with no space between them.